app/static/src/app/components/sensitivity/SensitivitySummaryPlot.vue

```diff
@@ -36,7 +36,7 @@ export default defineComponent({
         const plot = ref<null | HTMLElement>(null); // Picks up the element with 'plot' ref in the template
         const placeholderMessage = userMessages.sensitivity.notRunYet;
 
-        const batch = computed(() => store.state.sensitivity.result.batch);
+        const batch = computed(() => store.state.sensitivity.result?.batch);
```

Comment:
Opportunistic fix for error logged when switch Sensitivity plot to summary type when sensitivity has not yet been run.